### DISCUSSION TITLE
Mount NFS share mounted in subfolder to a mounted volume - error

### DISCUSSION THREAD

**Discussion by PelleFK** [containers/podman#20745] [Q&A] [answered]
HI,
I'm trying to run Apache with podman-compose. I'm also mounting local ./html folder to /var/www/htmlö in container under volumes. In a subfolder to ./html I have mounted a NFS share, called ./html/share. After I have mounted the NFS share I cannot start the container any mote. I get the error 
"Error: unable to start container "xxxxxxxxxx": lsetxattr /opt/apache/html/shared: operation not supported".
Should it be possible to mount like this?

I'm using Redhat 8 and SELinux are active.

**Comment by rhatdan** [comment, marked as answer]
:Z and :z is not supported on NFS shares.  Can you do it without specifying :Z?

**Reply by PelleFK** [reply]
Excelent! Thank you very very much. 
I have to fix the SELinux settings on my own but the container starts and can access the NFS share.
